draft-ietf-quic-recovery.md

```diff
@@ -272,9 +272,10 @@ SHOULD NOT be used to update RTT estimates if it does not newly acknowledge the
 largest acknowledged packet.
 
 An RTT sample MUST NOT be generated on receiving an ACK frame that does not
-newly acknowledge at least one ack-eliciting packet.  A peer does not send an
-ACK frame on receiving only non-ack-eliciting packets, so an ACK frame that is
-subsequently sent can include an arbitrarily large Ack Delay field.  Ignoring
+newly acknowledge at least one ack-eliciting packet.  A peer does usally not
```

Comment:
s/usally/usually/

Also, I'd place "usually" before does:

> A peer usually does not send [...]

Reply:
"does not usually" would also be fine.  But not in the middle.

Reply:
done